data/test/news/articles/c3wmq4d1y3wo.json

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
This is failing the dataValidator: `Error: Missing required property 'height' for ':article:content:model:blocks:video:model:blocks:aresMedia:model:blocks:image:model:blocks:rawImage:model'
`